core/server/services/auth/authenticate.js

```diff
@@ -100,7 +101,10 @@ const authenticate = {
         )(req, res, next);
     },
 
-    authenticateAdminAPI: [session.safeGetSession, session.getUser]
+    // ### v2 API auth middleware
+    authenticateAdminAPI: [session.safeGetSession, session.getUser],
```

Comment:
In order to keep the rebase simple i've left this. Once it's merged i'll open a PR to rename this to `authenticateAdminApiSession`